controllers/jenkinsconfig/jenkinsconfig_controller.go

```diff
@@ -252,6 +256,10 @@ func (c *Controller) syncHandler(key string) error {
 		return err
 	}
 
+	// Wait some period to reload
+	klog.V(5).Infof("waiting %s to reload Jenkins configuration", c.ReloadCasCPeriod.String())
```

Comment:
forget the unit of the time? minute? hour? or just second?

Reply:
Please see the output [above comment](https://github.com/kubesphere-sigs/ks-devops/pull/75#issuecomment-887343523).

```go
waiting 1m10s to reload Jenkins configuration
```

Reply:
got it.